### PR TITLE
Add observability example

### LINKED ISSUE #297: Add observability example
Thought it'd be good to add a little example project to demonstrate Camel Quarkus observability extensions:

* MicroProfile Health
* MicroProfile Metrics
* OpenTracing

### PR DESCRIPTION
**Pull request by jamesnetherton**
fixes #297

### CODE REVIEW

File: examples/observability/README.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+This example project demonstrates how to add support for metrics, health checks and distributed tracing.
+
+=== Running
+
+[source]
+----
+$ mvn clean compile quarkus:dev -DnoDeps
+----
+
```
Comment (ppalaga) [inline examples/observability/README.adoc]:
```suggestion
This compiles the project and starts the Quarkus tooling in the https://quarkus.io/guides/maven-tooling#development-mode[develpment mode].

To build a https://quarkus.io/guides/maven-tooling#building-a-native-executable[native executable] run the following (Docker required)

[source]
￼----
￼$ mvn clean package -Dnative
￼----

and find the native executable in the `target` directory.

```

Comment (jamesnetherton) [reply examples/observability/README.adoc]: Thanks, I've now added this to info the README.

**Comment by davsclaus** [comment]
@jamesnetherton ah I also added an example, do you mind resolving the examples/pom.xml conflict

**Comment by jamesnetherton** [comment]
Rebased & resolved conflicts.

**Pull request merged**